File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.kt
class Solution {
    fun solution(rsp: String): String {
        var ans = ""
        
        rsp.forEach {
            ans += when (it) {
                '0' -> '5'
                '2' -> '0'
                else -> '2'
            }
        }
        
        return ans
    }
}

File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string rsp) {
    string answer = "";
    
    for (char c : rsp) {
        if (c == '2') {
            answer += '0';
        } else if (c == '5') {
            answer += '2';
        } else {
            answer += '5';
        }
    }
    
    return answer;
}